_includes/head.html

```diff
@@ -35,7 +35,10 @@
                                                 `W###@#N#/b
                                                  .#mj#X~YM#
                                                  ~`#WdD~  Y!
-                                                 `'_|#f--          built by Presidential Innovation Fellows (http://pif.gov)
+                                                 `'_|#f--
+
+  Like code? Want to make America more awesome? Apply to be a Presidential
+  Innovation Fellow: https://apply.pif.gov/
```

Comment:
Cool — thanks for the line width limiting :-)

I would suggest keeping "Presidential Innovation Fellow" on the same line though, just for continuity.